src/js/helpers/timeRangeHelper.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * timeRangeHelper.js
+ * Created by Lizzie Salita 3/8/18
+ */
+
+import moment from 'moment';
+
+export const convertDatesToRange = (startDate, endDate) => {
+    if ((startDate && endDate) && (moment.isMoment(startDate) && moment.isMoment(endDate))) {
+        const duration = moment.duration(endDate.diff(startDate));
+        const years = duration.years();
+        const months = duration.months();
+        let yearString = '';
+        let monthString = '';
+
+        if (!isNaN(years)) {
```

Comment:
What is the condition `!isNaN(years)` is trying to catch? If it is to detect <1 year, wouldn't `years` be 0?